core-chart/values.yaml

```diff
@@ -57,6 +57,10 @@ status_agent:
 # Determine if the Post Create Hooks should be installed by the chart
 InstallPCHs: true
 
+# Transport controller bundling parameters
+transport_controller:
+    max_num_wrapped: 9223372036854775807
```

Comment:
I recommend changing the number here to 512000, because the current value is far larger than is of any actual use and nobody (that I know of) has this value in their head (despite it being very simple in binary). See also #2431 .